src/index.ts

```diff
@@ -42,6 +43,8 @@ export default class Dalai {
       process.env.USE_PYTHON_ENV === "1";
     this.modelsPath =
       process.env.MODELS_PATH ?? path.resolve(this.llamaPath, "models");
+    this.tempPromptsPath =
+      process.env.TEMP_PROMPTS_PATH ?? path.resolve(os.homedir(), "prompt-tmp");
```

Comment:
This is kind of a nitpick, but why add -tmp at the end when it's clearly in the /tmp folder. I think a better name would be dalai-prompts

Reply:
Sorry ignore, in the GitHub issue you used os.tmpdir...